Python/Product/Analysis/Interpreter/Pep440Version.cs

```diff
@@ -20,10 +20,9 @@
 using System.Linq;
 using System.Text;
 using System.Text.RegularExpressions;
-using System.Threading.Tasks;
 
-namespace Microsoft.PythonTools.EnvironmentsList {
-    struct Pep440Version : IComparable<Pep440Version>, IEquatable<Pep440Version> {
+namespace Microsoft.PythonTools.Interpreter {
+    public struct Pep440Version : IComparable<Pep440Version>, IEquatable<Pep440Version> {
```

Comment:
Renamed to PackageVersion to match PackageSpec

---
In reply to: [75514490](https://github.com/Microsoft/PTVS/pull/1536#discussion_r75514490) [](ancestors = 75514490)